docs/core-concepts/atomic-swap-htlc.md

```diff
@@ -0,0 +1,133 @@
+---
+id: atomic-swap-htlc
+title: Atomic Swaps using HTLCs
+sidebar_label: Atomic Swaps using HTLCs
+---
+
+[Atomic Swap](#atomic-swap)s using [HTLC](#htlc)s defines a solution using timelocks and hashlocks for exchanging two digital assets without the need of a trusted third party.
+
+Prior to the actual swap execution there is usually a [Negotiation](negotiation.md) phase where the two parties agree on the ledgers, assets and amounts to be swapped.
+
+Assuming that Alice has ETH on Ethereum and Bob has BTC on Bitcoin, Alice wants Bob's BTC and Bob wants Alice's ETH, the execution of a swap would be like this:
+
+1. Alice generates a secret and hashes it. She sends to Bob:
+    1. Hash of the secret. 
+    1. Her refund address on Ethereum.
+    1. Her redeem address on Bitcoin.
+    1. Proposed expiry times on the for the [HTLC](#htlc)s.
+1. Bob receives the swap proposed by Alice and sends back:
+    1. His refund address on Bitcoin.
+    1. His redeem address on Ethereum.
+1. Alice has to fund first, she deploys an Ethereum HTLC transferring the agreed amount of ETH into the contract.
+1. Bob monitors the Ethereum ledger and notices that Alice has deployed the HTLC funding the ETH. 
+He now sends a Bitcoin transaction into the Bitcoin network that is guarded by the script-hash of the Bitcoin HTLC (Bitcoin script).
+1. Alice monitors the Bitcoin ledger and notices that Bob has sent the funding transaction. She can now redeem the Bitcoin by spending the Bitcoin by revealing the secret.
```

Comment:
I had to think about this, and correct me if I'm wrong, but I think the asset 'bitcoin' would be spelled with a small 'b'.  The same way we write 'ten dollars' not 'ten Dollars'.  Same for ether.

Reply:
'spelled'/'spelt'  hmmm? - conjugating English verbs is a punish.

Reply:
> This means that you should say "spelled" in US English and you can use both "spelt" and "spelled" in UK/Australian English. Spelled vs spelt: In American English, spelt primarily refers to the hardy wheat grown mostly in Europe, and the verb spell makes spelled in the past tense and as a past participle